x/erc20/keeper/keeper_test.go

```diff
@@ -146,6 +146,31 @@ func (suite *KeeperTestSuite) TestDenomContractMap() {
 				suite.Require().Error(err)
 			},
 		},
+		{
+			"success, delete contract",
+			func() {
+				keeper := suite.app.Erc20Keeper
+				r := keeper.DeleteContractForDenom(suite.ctx, denom1)
+				suite.Require().Equal(r, false)
+				err := keeper.SetContractForDenom(suite.ctx, denom1, externalContract)
+				suite.Require().NoError(err)
+				r = keeper.DeleteContractForDenom(suite.ctx, denom1)
+				suite.Require().Equal(r, true)
+			},
+		},
+		{
+			"success, get contracts",
```

Comment:
maybe it is better to rename it to `success, multiple denoms map to different contracts`